docs/installation.rst

```diff
@@ -46,12 +46,6 @@ Pip
 
       $ python3 -m pip install molecule ansible-core
 
-.. warning::
```

Comment:
Add a `.. versionremoved:: 3.6.0` so that the absence is documented.

Reply:
I am not going to add a new sphinx plugin for this but I kept it as a standard warning.